pages_unit_tests/opening-hours.test.js

```diff
@@ -0,0 +1,364 @@
+        "InputField#opening_hours_tuesday"
+      );
+      expect(OpeningHoursField.props().input.defaultValue).toBe("default");
+    });
+    it("Wednesday input field gets given the correct default value", () => {
+      const cumulativeFullAnswers = {
+        opening_day_wednesday: "Wednesday",
+        opening_hours_wednesday: "default"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={testValidatorErrors}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find(
+        "InputField#opening_hours_wednesday"
+      );
+      expect(OpeningHoursField.props().input.defaultValue).toBe("default");
+    });
+
+    it("Thursday input field gets given the correct default value", () => {
+      const cumulativeFullAnswers = {
+        opening_day_thursday: "Thursday",
+        opening_hours_thursday: "default"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={testValidatorErrors}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find(
+        "InputField#opening_hours_thursday"
+      );
+      expect(OpeningHoursField.props().input.defaultValue).toBe("default");
+    });
+    it("Friday input field gets given the correct default value", () => {
+      const cumulativeFullAnswers = {
+        opening_day_friday: "Friday",
+        opening_hours_friday: "default"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={testValidatorErrors}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find("InputField#opening_hours_friday");
+      expect(OpeningHoursField.props().input.defaultValue).toBe("default");
+    });
+    it("Saturday input field gets given the correct default value", () => {
+      const cumulativeFullAnswers = {
+        opening_day_saturday: "Saturday",
+        opening_hours_saturday: "default"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={testValidatorErrors}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find(
+        "InputField#opening_hours_saturday"
+      );
+      expect(OpeningHoursField.props().input.defaultValue).toBe("default");
+    });
+    it("Sunday input field gets given the correct default value", () => {
+      const cumulativeFullAnswers = {
+        opening_day_sunday: "Sunday",
+        opening_hours_sunday: "default"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={testValidatorErrors}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find("InputField#opening_hours_sunday");
+      expect(OpeningHoursField.props().input.defaultValue).toBe("default");
+    });
```

Comment:
I would add one case which tests more than one day at once with different default values. This gives confidence that no copy/paste mistakes have been made. From a white-box perspective, we know this not to be the case here. From a black-box perspective, it's good to cover in case of refactoring.